code/get_analysis.py

```diff
@@ -0,0 +1,81 @@
+import helper
+import logging
+from telebot import types
+import matplotlib.pyplot as plt
+import time
+import numpy as np
+
```

Comment:
Generates blank image plots if there are no recorded expenses or budgets set. 
This is due to change in the implementation of the /delete command. Please pull the latest code and apply relevant changes/checks to rectify this issue.

Reply:
@shashank-madan there are checks to see if spend is 0

Reply:
Will have to modify the helper functions to check if category budget/ overall budget is available. As delete clears the entry, but does not set it to null. So isCategoryBudgetAvailable will return true, even if it is an empty dict. It will be the same for the other similar helper functions.

Reply:
Will check for 0 then.

Reply:
will fix in https://github.com/aditikilledar/dollar_bot_SE23/issues/35